### PR TITLE
[Kotlin SDK] doc update

### PR DESCRIPTION
**Pull request by astinz**
### Documentation Update

This pull request updates the documentation for `Kaptos`. The changes include:

- Reflecting the current API
- Improving word choices for better clarity
- Updating the sample code

### CODE REVIEW

File: apps/nextra/pages/en/build/sdks/swift-sdk.mdx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+<Callout>
+  **AptosKit** is currently under development, please give feedback [here](https://github.com/mcxross/swift-aptos/issues)
+</Callout>
+
+## Installation
+
+To install **AptosKit**, add the following to your `Package.swift` file:
+
+```swift
```
Comment (hariria) [inline apps/nextra/pages/en/build/sdks/swift-sdk.mdx:25]:
```suggestion
```swift filename="Package.swift"
```

File: apps/nextra/pages/en/build/sdks/swift-sdk/quickstart.mdx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+This guide will walk you through the process of setting up **AptosKit**, and
+fetching data on the Aptos blockchain.
+
+<Steps>
+### Install the SDK
+
+**AptosKit** is available as a Swift package. To add it to your project,
+add the following to your `Package.swift` file:
+
+```swift
```
Comment (hariria) [inline apps/nextra/pages/en/build/sdks/swift-sdk/quickstart.mdx:18]:
```suggestion
```swift filename="Package.swift"
```

File: apps/nextra/pages/en/build/sdks/swift-sdk/quickstart.mdx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  let config = ClientConfig(
+  followRedirects: true,
+  agent: "AptosClient",
+  likeAgent: nil,
+  requestTimeout: 5000,
+  retryOnServerErrors: 3,
+  maxRetries: 5,
+  cache: false,
+  proxy: nil
+  )
```
Comment (hariria) [inline apps/nextra/pages/en/build/sdks/swift-sdk/quickstart.mdx:47]: Nit: fix indenting

File: apps/nextra/pages/en/build/sdks/swift-sdk/quickstart.mdx [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+dependencies: [
+  .package(url: "https://github.com/mcxross/swift-aptos.git", .upToNextMajor(from: <version>))
+]
+```
+
+### Import the SDK
+
+Import the SDK in your Swift file:
+
+```swift
```
Comment (hariria) [inline apps/nextra/pages/en/build/sdks/swift-sdk/quickstart.mdx:28]: Add filename to codeblock (same as above)

File: apps/nextra/pages/en/build/sdks/swift-sdk/quickstart.mdx [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+  network: .devnet,
+  fullNode: nil,
+  faucet: nil,
+  indexer: nil,
+  client: nil,
+  clientConfig: config,
+  fullNodeConfig: nil,
+  indexerConfig: nil,
+  faucetConfig: nil
+  )
```
Comment (hariria) [inline apps/nextra/pages/en/build/sdks/swift-sdk/quickstart.mdx:66]: Nit: Indenting / formatting

File: apps/nextra/pages/en/build/sdks/swift-sdk/quickstart.mdx [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+```
+
+### Create the `AptosConfig` object
+
+```swift
+  let aptosConfig = AptosConfig(settings: aptosSettings)
```
Comment (hariria) [inline apps/nextra/pages/en/build/sdks/swift-sdk/quickstart.mdx:72]: Nit: Indenting / formatting

File: apps/nextra/pages/en/build/sdks/swift-sdk/quickstart.mdx [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+```
+
+### Create the `Aptos` object
+
+This object is used to interact with the Aptos blockchain. It serves as the
+entry point for all interactions with the blockchain.
+
+```swift
+  let aptos = Aptos(config: aptosConfig, graceFull: false)
```
Comment (hariria) [inline apps/nextra/pages/en/build/sdks/swift-sdk/quickstart.mdx:81]: Nit: Indenting / formatting

File: apps/nextra/pages/en/build/sdks/swift-sdk/quickstart.mdx [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+```
+
+### Fetch the chain ID
+
+```swift
+  let chainId = try await aptos.getChainId()
```
Comment (hariria) [inline apps/nextra/pages/en/build/sdks/swift-sdk/quickstart.mdx:87]: Nit: Indenting / formatting

File: apps/nextra/pages/en/build/sdks/swift-sdk/quickstart.mdx [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+This object is used to interact with the Aptos blockchain. It serves as the
+entry point for all interactions with the blockchain.
+
+```swift
+  let aptos = Aptos(config: aptosConfig, graceFull: false)
+```
+
+### Fetch the chain ID
+
+```swift
```
Comment (hariria) [inline apps/nextra/pages/en/build/sdks/swift-sdk/quickstart.mdx:86]: Add filename to codeblock

File: apps/nextra/pages/en/build/sdks/swift-sdk/quickstart.mdx [hunk 10]
```diff
@@ -0,0 +1,90 @@
...
+  let aptosConfig = AptosConfig(settings: aptosSettings)
+```
+
+### Create the `Aptos` object
+
+This object is used to interact with the Aptos blockchain. It serves as the
+entry point for all interactions with the blockchain.
+
+```swift
```
Comment (hariria) [inline apps/nextra/pages/en/build/sdks/swift-sdk/quickstart.mdx:80]: Add filename to codeblock

File: apps/nextra/pages/en/build/sdks/swift-sdk/quickstart.mdx [hunk 11]
```diff
@@ -0,0 +1,90 @@
...
+  clientConfig: config,
+  fullNodeConfig: nil,
+  indexerConfig: nil,
+  faucetConfig: nil
+  )
+```
+
+### Create the `AptosConfig` object
+
+```swift
```
Comment (hariria) [inline apps/nextra/pages/en/build/sdks/swift-sdk/quickstart.mdx:71]: Add filename to codeblock

File: apps/nextra/pages/en/build/sdks/swift-sdk/quickstart.mdx [hunk 12]
```diff
@@ -0,0 +1,90 @@
...
+  proxy: nil
+  )
+```
+
+### Create the `AptosSettings` object
+
+This object is used to configure the Aptos network connection. You can set `network`,
+`fullnode`, and `faucet` properties.
+
+```swift
```
Comment (hariria) [inline apps/nextra/pages/en/build/sdks/swift-sdk/quickstart.mdx:55]: Add filename to codeblock

File: apps/nextra/pages/en/build/sdks/swift-sdk/quickstart.mdx [hunk 13]
```diff
@@ -0,0 +1,90 @@
...
+  import AptosKit
+```
+
+### Create the `ClientConfig` object
+
+This object is used to configure the client behavior. You can set `maxRetries`,
+`requestTimeout`, and `retryOnServerErrors` properties.
+
+```swift
```
Comment (hariria) [inline apps/nextra/pages/en/build/sdks/swift-sdk/quickstart.mdx:37]: Add filename to codeblock

**Review by hariria — COMMENTED** [review]
Will let @gregnazario take a look as well

File: apps/nextra/pages/en/build/sdks/swift-sdk/quickstart.mdx [hunk 14]
```diff
@@ -0,0 +1,90 @@
...
+      network: .devnet,
+      fullNode: nil,
+      faucet: nil,
+      indexer: nil,
+      client: nil,
+      clientConfig: config,
+      fullNodeConfig: nil,
+      indexerConfig: nil,
+      faucetConfig: nil
+  )
```
Comment (hariria) [inline apps/nextra/pages/en/build/sdks/swift-sdk/quickstart.mdx:66]: Fix indenting / formatting

Comment (astinz) [reply apps/nextra/pages/en/build/sdks/swift-sdk/quickstart.mdx:66]: # Expansion

Unless there’s a specific required indentation for the documentation, please share it. However, this formatting follows Xcode’s style, which is generally considered the recommended approach:

CleanShot 2024-07-23 at 11 12 10@2x

## `pnpm fmt`
Not quite sure of the above command in the [docs](https://github.com/aptos-labs/developer-docs?tab=readme-ov-file#debugging), but it barely does anything.

**Comment by WGB5445** [comment]
Thank you very much for your contribution. Currently, we have noticed that other organizations have also contributed Swift SDKs. We want to ensure that everyone’s efforts are not overlooked, and I truly appreciate your work. We can explore the possibility of showcasing different Swift SDKs on separate pages, which would allow us to present everyone’s contributions effectively and to everyone’s satisfaction. 
So could the Swift portion be merged through a separate PR? 

Thank you again for your valuable contributions.

**Comment by igor-p** [comment]
@gregnazario @WGB5445 Are we still moving ahead with this? Or do we want to split out the Swift portion and leave the rest?

**Comment by astinz** [comment]
Hey @igor-p,

I suggest keeping this as a subtopic under the **Kotlin SDK**, titled: “**Kotlin SDK for iOS Developers.**” . The latest deployment can serve as a ref. 

@WGB5445 may continue with the native Swift SDK. This will add variety to the clients.

**Comment by gregnazario** [comment]
> @gregnazario @WGB5445 Are we still ok with this effort? Or do we want to split out the Swift portion and leave the rest?

We can leave that section, once this PR is done, we will move it under Community SDKs

**Comment by gregnazario** [comment]
@astinz let's get this completed to some state that works for you, and then we're moving it under Community SDKs.

**Comment by astinz** [comment]
@gregnazario proceed, everything looks good on my end...

**Review by gregnazario — APPROVED** [review]

**Pull request merged**